File: res/material/sqlite/13天-SQLite/备课代码/03-谓词演练/03-谓词演练/Person.m
//
//  Person.m
//  03-谓词演练
//
//  Created by 刘凡 on 15/12/20.
//  Copyright © 2015年 joyios. All rights reserved.
//

#import "Person.h"

@implementation Person

- (NSString *)description {
    return [NSString stringWithFormat:@"%@ ~ %@ ~ %zd", _name, _title, _age];
}

@end

File: practices/day42-swiftbeginning/day42-swiftbeginning/Person.swift


//
//  Person.swift
//  day42-swiftbeginning
//
//  Created by apple on 15/12/3.
//  Copyright © 2015年 yf. All rights reserved.
//

import UIKit

class Person: NSObject {
    var name:String?
    var age:Int=0
    
    init(dict:[String:AnyObject]) {
        super.init()
        setValuesForKeysWithDictionary(dict)
    }
    
    override func setValue(value: AnyObject?, forUndefinedKey key: String) {
        
    }
    
    
}
